linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml

```diff
@@ -27,11 +27,14 @@ identifiers:
     cce@rhel7: CCE-80192-8
     cce@rhel8: CCE-84275-7
     cce@rhel9: CCE-83995-1
+    cce@sle12: CCE-92259-1
+    cce@sle15: CCE-91375-6
 
 references:
     cis-csc: 1,11,12,13,14,15,16,18,3,4,5,6,8,9
     cis@alinux3: 4.2.1.6
-    cis@rhel7: 4.2.1.5
+    cis@rhel7: 4.2.1.
```

Comment:
Is it a typo or is the id incomplete?

Reply:
Hello @marcusburghardt 
It is a typo. I have done the expected correction.
Have a nice day
Rumen